.github/workflows/main-build.yml

```diff
@@ -5,6 +5,7 @@ on:
     workflows: ["Path tests"]
     types:
       - completed
+    branches: ["main"]
```

Comment:
Need to add this to the release workflow as well.

Also can we use 

```yaml
branches:
    - main
```
    syntax for consistency?